packages/input_schema/src/types.ts

```diff
@@ -18,6 +18,9 @@ export type StringFieldDefinition = CommonFieldDefinition<string> & {
     enum?: readonly string[]; // required if editor is 'select'
     enumTitles?: readonly string[]
     isSecret?: boolean;
+    // used for 'datepicker' editor, isAbsolute is considered with default value true
+    isAbsolute?: boolean;
```

Comment:
Why do we have two fields isAbsolute and isRelative, is it isRelative enaght?
if the field is isRelative it is relative otherwise it is absolute, the absolute is default value anyway

Reply:
Because it can accept both formats, so if only `isRelative: true` is specified it accepts both unless `isAbsolute` is explicitly set to `false`

Reply:
Using better naming for param isRelative and isAbsolute are confusing, allowAbsolute and allowRelative?